CMakeLists.txt

```diff
@@ -9,12 +9,39 @@ Emscripten.cmake"
     )
 endif()
 
+# Extract the project name & version from package.json
+if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/package.json")
+    file(READ "${CMAKE_CURRENT_SOURCE_DIR}/package.json" PACKAGE_JSON_CONTENT)
```

Comment:
How about `CLP_FFI_JS_PACKAGE_JSON_CONTENT`? (To follow our convention of prefixing local variables with the project name.)